production-marathon/README.md

```diff
@@ -47,4 +47,4 @@ In order to deploy the service application, issue the following command from the
 	source ./marathon.env ; \
 		cat marathon.json.b \
 		| perl -pe 's/\$\{(\w+)\}/(exists $ENV{$1} && length $ENV{$1} > 0 ? $ENV{$1} : "NOT_SET_$1")/eg' \
-		| curl -H "Content-Type: application/json" -d @- http://localhost:8080/v2/groups/
+		| curl -H "Content-Type: application/json" -d @- http://localhost:8080/v2/groups/caf
```

Comment:
This command was updated because the original command could not deploy another application into the root caf group without destroying existing apps running in the caf group first. Problem was found by me and the decision to make this change was determined by Greg and Dermot.